src/Moryx.Model.Sqlite/SqliteModelConfigurator.cs

```diff
@@ -38,6 +38,11 @@ protected override DbCommand CreateCommand(string cmdText, DbConnection connecti
         /// <inheritdoc />
         public override Task DeleteDatabase(IDatabaseConfig config)
         {
+            using (var sqliteConnection = (SqliteConnection)CreateConnection(config))
```

Comment:
You created the sqliteconnection here but never used it. ClearAllPools is a static method you don't need a surrounding using statemant for 🙃